src/main/java/com/rockset/client/ApiClient.java

```diff
@@ -360,9 +361,11 @@ public ApiClient setDebugging(boolean debugging) {
             if (debugging) {
                 loggingInterceptor = new HttpLoggingInterceptor();
                 loggingInterceptor.setLevel(Level.BODY);
-                httpClient.interceptors().add(loggingInterceptor);
+                setHttpClient(getHttpClient().newBuilder().addInterceptor(loggingInterceptor).build());
```

Comment:
the swagger codegen was trying to mutate immutable objects. Had to change it to use new builder